packages/lesswrong/lib/rateLimits/utils.ts

```diff
@@ -0,0 +1,180 @@
+import moment from "moment"
+import { getDownvoteRatio } from "../../components/sunshineDashboard/UsersReviewInfoCard"
```

Comment:
importing from a component to a lib file feels like it's asking for circular dependency issues at some point, but probably fine for now